packages/select/docs/MenuLoadingSkeleton.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import React from "react"
+import { Meta, StoryFn, StoryObj } from "@storybook/react"
+import { ComponentDocsTemplate } from "../../../storybook/components/DocsContainer"
+import { StickerSheet } from "../../../storybook/components/StickerSheet"
+import { MenuLoadingSkeleton } from "../src/FilterMultiSelect/components/MenuLayout/MenuLoadingSkeleton/index"
+
+const meta = {
+  tags: ["autodocs"],
+  title: "Components/Filter Multi-Select/Menu Loading Skeleton",
+  component: MenuLoadingSkeleton,
+  args: {
+    isAnimated: false,
+  },
+  parameters: {
+    installation: [
+      "npm install @kaizen/select",
+      "import { FilterMultiSelect } from `@kaizen/select`",
+    ],
+    chromatic: { disable: false },
```

Comment:
This is the wrong place to put this because we don't want to be sending every story from this file as a snapshot (especially not an animated one). If you're not making a sticker sheet, then you need to add this param only to the `Playground` in your case.

Reply:
ohhh right, makes sense!